package.json

```diff
@@ -1,25 +1,110 @@
 {
-  "name": "ara",
-  "displayName": "Ara Highlighting",
-  "description": "Ara Programming Language Highlighting",
-  "version": "0.0.1",
-  "engines": {
-    "vscode": "^1.74.0"
-  },
-  "categories": [
-    "Programming Languages"
-  ],
-  "contributes": {
-    "languages": [{
-      "id": "ara",
-      "aliases": ["Ara", "ara"],
-      "extensions": [".ara"],
-      "configuration": "./language-configuration.json"
-    }],
-    "grammars": [{
-      "language": "ara",
-      "scopeName": "source.ara",
-      "path": "./syntaxes/ara.json"
-    }]
-  }
+    "name": "ara.vscode",
+    "displayName": "ARA",
```

Comment:
```suggestion
    "displayName": "Ara Highlighting",
```

Reply:
My reasoning behing this was that the extension would eventually do more than highlighting.

Reply:
no, this would just be for highlighting really, LSP extension would be it's own.

Reply:
later we can create an extension pack that contains both.

Reply:
Fair enough.